File: src/test/java/com/sidgate/HandTest.java
package com.sidgate;

import com.sidgate.deck.Card;
import com.sidgate.deck.Hand;
import org.junit.Test;

import static java.util.Arrays.asList;
import static org.junit.Assert.assertTrue;

public class HandTest {

	@Test
	public void testHandCards() {
		Hand hand = new Hand(asList(new Card("H5"), new Card("H6")));
		assertTrue(hand.contains(new Card("H5")));
	}

	@Test
	public void testHandCardsByText() {
		Hand hand = new Hand("H5,H6");
		assertTrue(hand.contains(new Card("H5")));
	}
}

File: src/test/kotlin/sidgate/HandTest.kt
package sidgate

import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test
import sidgate.deck.Card
import sidgate.deck.Hand

class HandTest {
    @Test
    fun testHandCards() {
        val hand = Hand(listOf(Card("H5"), Card("H6")))
        assertTrue(hand.contains(Card("H5")))
    }

    @Test
    fun testHandCardsByText() {

        val hand = Hand(cards("H5,H6"))
        assertTrue(hand.contains(Card("H5")))
    }

    private fun cards(list: String) = list.split(",").map { Card(it) }
}